contrib/gitian-build.sh

```diff
@@ -260,153 +260,155 @@ popd
+    pushd ./gitian-builder
+    mkdir -p inputs
+    wget -N -P inputs $osslPatchUrl
+    wget -N -P inputs $osslTarUrl
+    make -C ../pivx/depends download SOURCES_PATH=`pwd`/cache/common
 
-        if [[ $commitFiles = true ]]
-        then
-	    # Commit to gitian.sigs repo
-            echo ""
-            echo "Committing ${VERSION} Unsigned Sigs"
-            echo ""
-            pushd gitian.sigs
-            git add ${VERSION}-linux/${SIGNER}
-            git add ${VERSION}-aarch64/${SIGNER}
-            git add ${VERSION}-win-unsigned/${SIGNER}
-            git add ${VERSION}-osx-unsigned/${SIGNER}
-            git commit -a -m "Add ${VERSION} unsigned sigs for ${SIGNER}"
-            popd
-        fi
+    # Linux
+    if [[ $linux = true ]]
+    then
+        echo ""
+        echo "Compiling ${VERSION} Linux"
+        echo ""
+        ./bin/gbuild -j ${proc} -m ${mem} --commit pivx=${COMMIT} --url pivx=${url} ../pivx/contrib/gitian-descriptors/gitian-linux.yml
+        ./bin/gsign -p $signProg --signer $SIGNER --release ${VERSION}-linux --destination ../gitian.sigs/ ../pivx/contrib/gitian-descriptors/gitian-linux.yml
+        mv build/out/pivx-*.tar.gz build/out/src/pivx-*.tar.gz ../pivx-binaries/${VERSION}
+    fi
+    # Windows
+    if [[ $windows = true ]]
+    then
+        echo ""
+        echo "Compiling ${VERSION} Windows"
+        echo ""
+        ./bin/gbuild -j ${proc} -m ${mem} --commit pivx=${COMMIT} --url pivx=${url} ../pivx/contrib/gitian-descriptors/gitian-win.yml
+        ./bin/gsign -p $signProg --signer $SIGNER --release ${VERSION}-win-unsigned --destination ../gitian.sigs/ ../pivx/contrib/gitian-descriptors/gitian-win.yml
+        mv build/out/pivx-*-win-unsigned.tar.gz inputs/pivx-win-unsigned.tar.gz
+        mv build/out/pivx-*.zip build/out/pivx-*.exe ../pivx-binaries/${VERSION}
+    fi
+    # Mac OSX
+    if [[ $osx = true ]]
+    then
+        echo ""
+        echo "Compiling ${VERSION} Mac OSX"
+        echo ""
+        ./bin/gbuild -j ${proc} -m ${mem} --commit pivx=${COMMIT} --url pivx=${url} ../pivx/contrib/gitian-descriptors/gitian-osx.yml
+        ./bin/gsign -p $signProg --signer $SIGNER --release ${VERSION}-osx-unsigned --destination ../gitian.sigs/ ../pivx/contrib/gitian-descriptors/gitian-osx.yml
+        mv build/out/pivx-*-osx-unsigned.tar.gz inputs/pivx-osx-unsigned.tar.gz
+        mv build/out/pivx-*.tar.gz build/out/pivx-*.dmg ../pivx-binaries/${VERSION}
+    fi
+    # AArch64
+    if [[ $aarch64 = true ]]
+    then
+        echo ""
+        echo "Compiling ${VERSION} AArch64"
+        echo ""
+        ./bin/gbuild -j ${proc} -m ${mem} --commit pivx=${COMMIT} --url pivx=${url} ../pivx/contrib/gitian-descriptors/gitian-aarch64.yml
+        ./bin/gsign -p $signProg --signer $SIGNER --release ${VERSION}-aarch64 --destination ../gitian.sigs/ ../pivx/contrib/gitian-descriptors/gitian-aarch64.yml
+        mv build/out/pivx-*.tar.gz build/out/src/pivx-*.tar.gz ../pivx-binaries/${VERSION}
+        #popd
```

Comment:
Nit: I don't think this comment needs to stay here

Reply:
I have ammended the comment and arranged the commit history  as told.

Thank you